src/fswAlgorithms/_GeneralModuleFiles/measurementModels.h

```diff
@@ -19,35 +19,57 @@
 
 #include <Eigen/Core>
 #include "architecture/utilities/rigidBodyKinematics.hpp"
+#include "fswAlgorithms/_GeneralModuleFiles/stateModels.h"
 
 #ifndef FILTER_MEAS_MODELS_H
 #define FILTER_MEAS_MODELS_H
 
 /*! @brief Container class for measurement data and models */
-class Measurement{
+class MeasurementModel{
 public:
-    Measurement() = default;
+    MeasurementModel();
```

Comment:
:large_orange_diamond: It's a clearer expression of intent to `default` these in the header, rather than the implementation file.

Reply:
done everywhere